HW08/ArraySizeException.java

```diff
@@ -0,0 +1,6 @@
+public class ArraySizeException extends RuntimeException{
```

Comment:
Not sure why this exception must be uncheckable. Please see the comment above and rewrite them.